### DISCUSSION TITLE
flink cdc 1.16.0 sink doris1.2.1，unique key模型，数据更新不及时

### DISCUSSION THREAD

**Discussion by pengtao211** [apache/doris#16307] [A - General / Q&A] [answered]
当修改一笔数据时，测试发现doris数据更新和checkpoint interval相关，只有做checkpoint时，doris表数据才会更新，对比1.1.2版本，sink doris参数取消了'sink.batch.size' = '0','sink.batch.interval' = '1'，这2个配置，1.1.2版本时是可以做到近实时更新，1.2.1版本怎么才能让doris近实时更新数据了？

**Comment by gnehil** [comment, marked as answer]
可以调小checkpoint的间隔

**Reply by pengtao211** [reply]
如果调小checkpoint的间隔，开启RocksDB增量快照要好些吧，减少做checkpoint的时间

**Reply by pengtao211** [reply]
我决定用StarRocks来代替doris了

**Reply by gnehil** [reply]
增量快照是可以的，但是每次快照应该不会保存特别大的状态
